Dockerfile

```diff
@@ -19,7 +19,6 @@ USER node
 WORKDIR /home/node
 COPY ./ ./
 RUN npm ci
-RUN npm run build-contracts
```

Comment:
build directory is listed in .dockerignore

Reply:
are you sure stuff will work without build-contracts? Some code depends on `build/contracts` directory for smart contract ABIs for sure.

Reply:
so umm ok, maybe remove from .gitignore then. I've probably added it into ignore because re-building causes lots of noise. This could be improved by removing the AST from the resulting files, probably some etherlime flags might do that.

Reply:
One option here would be to do `npm ci` that includes the devDependencies, then run build-contracts, then `npm prune --production` to delete the devDependencies (like etherlime).